src/main/kotlin/no/nav/syfo/job/VarselSender.kt

```diff
@@ -23,14 +28,11 @@ class VarselSender(
         log.info("Starter SendVarslerJobb")
 
         val varslerSendt = HashMap<String, Int>()
-        var varslerToSendToday = databaseAccess.fetchPlanlagtVarselByUtsendingsdato(LocalDate.now())
+        val varslerToSendToday = databaseAccess.fetchPlanlagtVarselByUtsendingsdato(LocalDate.now())
 
-        if (toggles.sendMerVeiledningVarslerBasedOnMaxDate) {
-            val varslerToSendTodayMonthMerVeiledning = databaseAccess.fetchPlanlagtMerVeiledningVarselByUtsendingsdato(LocalDate.now())
-            val allVarslerToSendTodayMerVeiledning = varslerToSendTodayMonthMerVeiledning.plus(getAllUnsentMerVeiledningVarslerLastMonth())
 
-            varslerToSendToday = mergePlanlagteVarsler(varslerToSendToday, allVarslerToSendTodayMerVeiledning)
-            log.info("Planlegger å sende ${allVarslerToSendTodayMerVeiledning.size} Mer veiledning varsler med utsending basert på maxdato")
+        if (toggles.toggleInfotrygdKafkaConsumer && toggles.toggleUtbetalingKafkaConsumer) {
+            merVeiledningVarselFinder.findMerVeiledningVarslerToSendToday()
```

Comment:
Tenkte du å ta bruk av denne i en annen PR?

Reply:
Hva mener du? Denne blir bruk, begge toggles er enabled i prod. Skulle vi ikke bare se hvor mange varsler kommer til å bli sendt av den nye løsningen?

Reply:
Trodde vi skulle kjøre på med ny løsning

Reply:
At vi tenker vi er good to go :)